sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_generated/tsp-location.yaml

```diff
@@ -0,0 +1,8 @@
+directory: specification/keyvault/Security.KeyVault.Administration
```

Comment:
A general question regarding typespec generation: Is it normal/recommended for the tsp-location.yaml file to be nested here and not top-level? I suppose if you can't specify an output directory, you'd need it here to have the generated code be located in `_generated`...So I'm wondering was your workflow to manually create this file here, then use `tsp-client` to generate?